README.md

```diff
@@ -28,6 +28,8 @@ Usage: bookmarker [options]
 			Default: Google Chrome Bookmarks
   -d, --description     Set a description for the rendered markdown.
 			Default: Generated by Bookmarker
+  -s, --sort            Set if the list is sorted.
+                        Default: false
```

Comment:
Just one last thing.

Can you put this "Default" at the same indent as the other ones?